qubesmanager/restore.py

```diff
@@ -38,6 +39,17 @@
 
 
 # pylint: disable=too-few-public-methods
+def parse_args() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+            '--log', action='store', default='WARNING',
```

Comment:
Default was "INFO" before, changing to "WARNING" will hide most progress info...

Reply:
Done